modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -147,6 +147,13 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
         else{
             $useProjects = true;
         }
+        // Get sites
+        $sitesCond = '';
+        if (!$user->hasPermission('access_all_profiles') 
```

Comment:
To clarify "choice of permission": There's also a permission named "violated_scans_view_allsites" in the schema.